variables.tf

```diff
@@ -37,4 +49,9 @@ variable "number_of_nodes" {
 variable "port" {
     description = "The port number of your ElastiCache Cluster. Redis uses '6379' and Memached uses '11211'. The default will be '6379' with Redis being the assumed engine"
     default = 6379
-}
\ No newline at end of file
+}
+
+variable "ec_subnet_groups" {
+    description = "A list of VPC subnet IDs."
+    default = ["subnet-7293103a", "subnet-7bf10c21", "subnet-de00b3b8"]
```

Comment:
These are the public subnets for the VPC, can you change them to the private ones please?